<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuesWort</class>
 <widget class="QDialog" name="NeuesWort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wörter hinzufügen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/icons/res/icons/add_box_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/res/icons/add_box_FILL0_wght400_GRAD0_opsz24.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_deutsch">
     <property name="text">
      <string>Deutsch:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txt_wort"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_fremdsprache">
     <property name="text">
      <string>Fremdsprache:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txt_fremdwort"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_definition">
     <property name="text">
      <string>Definition:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txt_definition"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_bemerkung">
     <property name="text">
      <string>Bemerkung:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txt_bemerkung"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>199</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_erfolg">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 85, 0);</string>
       </property>
       <property name="text">
        <string>Wort hinzugefügt!</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmd_hinzufuegen">
       <property name="text">
        <string>Hinzufügen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmd_fertig">
       <property name="text">
        <string>Fertig</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
